src/mlpack/bindings/R/CMakeLists.txt

```diff
@@ -178,7 +178,7 @@ if (BUILD_R_BINDINGS)
   string(REGEX REPLACE ",$" "" AUTHORS_R_OUT "${AUTHORS_R}")
   set(AUTHORS_R "${AUTHORS_R_OUT})")
 
-  configure_file(${CMAKE_SOURCE_DIR}/src/mlpack/bindings/R/mlpack/DESCRIPTION.in
+  configure_file(${CMAKE_CURRENT_SOURCE_DIR}/src/mlpack/bindings/R/mlpack/DESCRIPTION.in
```

Comment:
I am not 100% sure but I think this relative path will need to be `${CMAKE_CURRENT_SOURCE_DIR}/mlpack/DESCRIPTION.in`.  I suspect there are several places like this through the binding configuration code.